bridges/snowbridge/pallets/ethereum-client/src/config/mod.rs

```diff
@@ -1,35 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
-use snowbridge_beacon_primitives::merkle_proof::{generalized_index_length, subtree_index};
 use static_assertions::const_assert;
 
-/// Generalized Indices
-/// related to Merkle proofs
-/// get_generalized_index(BeaconState, 'block_roots')
-pub const BLOCK_ROOTS_INDEX: usize = 37;
-pub const BLOCK_ROOTS_SUBTREE_INDEX: usize = subtree_index(BLOCK_ROOTS_INDEX);
-pub const BLOCK_ROOTS_DEPTH: usize = generalized_index_length(BLOCK_ROOTS_INDEX);
-
-/// get_generalized_index(BeaconState, 'finalized_checkpoint', 'root')
-pub const FINALIZED_ROOT_INDEX: usize = 105;
-pub const FINALIZED_ROOT_SUBTREE_INDEX: usize = subtree_index(FINALIZED_ROOT_INDEX);
-pub const FINALIZED_ROOT_DEPTH: usize = generalized_index_length(FINALIZED_ROOT_INDEX);
-
-/// get_generalized_index(BeaconState, 'current_sync_committee')
-pub const CURRENT_SYNC_COMMITTEE_INDEX: usize = 54;
-pub const CURRENT_SYNC_COMMITTEE_SUBTREE_INDEX: usize = subtree_index(CURRENT_SYNC_COMMITTEE_INDEX);
-pub const CURRENT_SYNC_COMMITTEE_DEPTH: usize =
-	generalized_index_length(CURRENT_SYNC_COMMITTEE_INDEX);
-
-/// get_generalized_index(BeaconState, 'next_sync_committee')
-pub const NEXT_SYNC_COMMITTEE_INDEX: usize = 55;
-pub const NEXT_SYNC_COMMITTEE_SUBTREE_INDEX: usize = subtree_index(NEXT_SYNC_COMMITTEE_INDEX);
-pub const NEXT_SYNC_COMMITTEE_DEPTH: usize = generalized_index_length(NEXT_SYNC_COMMITTEE_INDEX);
-
-///  get_generalized_index(BeaconBlockBody, 'execution_payload')
-pub const EXECUTION_HEADER_INDEX: usize = 25;
-pub const EXECUTION_HEADER_SUBTREE_INDEX: usize = subtree_index(EXECUTION_HEADER_INDEX);
-pub const EXECUTION_HEADER_DEPTH: usize = generalized_index_length(EXECUTION_HEADER_INDEX);
+pub mod altair;
+pub mod electra;
```

Comment:
Config is now based on fork version, so add a module per fork specific settings.